packages/xo-server-sdn-controller/src/index.js

```diff
@@ -831,7 +831,7 @@ class SDNController extends EventEmitter {
   }
 
   async _hostUpdated(host) {
-    const newHost = find(this._newHosts, { $ref: host.$ref })
+    const newHost = this._newHosts.find(_ => _.$ref === host.$ref)
```

Comment:
Do you need to check this if `!host.enabled || host.PIFs.length === 0`?

Reply:
No, do you think I should split the tests in 2?
I did it at once to reduce the number of line but I can split it back.

Reply:
Yes please:

```js
let newHost
if (!host.enabled || host.PIFs.length === 0 || (newHost = thiss._newHosts.find(_ => _.$ref === host.$ref)) === undefined) {
```